### PR TITLE
[MBL-15763] - Implement missing today button schedule interaction test case

### PR DESCRIPTION
**Pull request by kdeakinstructure**
refs: MBL-15763
affects: Student
release note: none

implement today button interaction test case for k5 schedule page

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: apps/student/src/androidTest/java/com/instructure/student/ui/pages/SchedulePage.kt [hunk 1]
```diff
@@ -90,6 +90,18 @@ class SchedulePage : BasePage(R.id.schedulePage) {
         pager.swipeLeft()
     }
 
+    fun swipeDown() {
```
Comment (tamaskozmer) [inline apps/student/src/androidTest/java/com/instructure/student/ui/pages/SchedulePage.kt:93]: A bit confusing naming. (`swipeDown` does a `swipeUp`). I guess this was meant to be `scrollDown`.

**Review by tamaskozmer — APPROVED** [review]

File: apps/student/src/androidTest/java/com/instructure/student/ui/pages/SchedulePage.kt (hunk 1 above)
Comment (kdeakinstructure) [reply apps/student/src/androidTest/java/com/instructure/student/ui/pages/SchedulePage.kt:93]: Nice catch, thx!

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**